### ISSUE TITLE
Excel files exported but do not show associated with Excel in Firefox

### ISSUE THREAD

**Issue by desuraghu** [ghiscoding/Angular-Slickgrid#500]
### Your Environment

| Software                | Version(s) |
| -------------------- | -------- |
| Angular                  | 7.2.4        | 
| Angular-Slickgrid   | 2.17.11        | 
| TypeScript              | 3.2        | 

### Describe the Bug
Exporting data through excel export service. In chrome download is working fine.
But in firefox the file is not associated with Excel, instead it has been marked as zip.

### Steps to Reproduce
ExcelExport in Firefox

### Expected Behavior
exported file should be associated with Excel

### Possible Solution
During blob creation, we need to specify the mime type of the file. So that browser will detect the type and it will show the correct associated application.

**Comment by ghiscoding** [comment]
I don't typically deal with issues closed by the bot but I see this as a valid issue, just make sure to fill in all required section of the issue template next time if you don't want the bot to auto-close it.

As for the issue with Firefox, I can reproduce but I'm not sure how to fix it since the Blob is not created in my lib but rather by the `ExcelBuilder` external lib. So that would need more investigation,

**Comment by desuraghu** [comment]
Thanks for looking into it. I was not sure, which data i have missed it. Will try to follow the guidelines next time.
How about creating a new blob with the returned object. I am sure this is not a good idea to clone the blob.

**Comment by ghiscoding** [comment]
The template required fields all have the `<!-- REQUIRED ...>` and I think you missed `Current Behavior`, it's ok not a big deal, like I said it's mostly in place to close issues that just start with "Please help..." without providing much info and I got tired of those and added the auto-close bot and now I get better detailed issues since then ;)

As for the issue itself, I found how to pass the mime type to the external lib in the 2nd argument for the options, the following seems to work
```ts
const mimeType = this._fileFormat === FileType.xls ? 'application/vnd.ms-excel' : 'application/vnd.openxmlformats-officedocument.spreadsheetml.sheet ';
ExcelBuilder.Builder.createFile(this._workbook, { type: 'blob', mimeType });
```

I think that is the correct type for `xlsx` but perhaps you could validate that, in my Firefox it now shows this popup and it opens correctly now

![image](https://user-images.githubusercontent.com/643976/84929534-efef1a80-b09d-11ea-9a73-a06a9b0b5f24.png)

**Comment by desuraghu** [comment]
Yes mime type is correct for both xls and xlsx.

**Comment by ghiscoding** [comment]
This is now fixed in latest version [2.19.x](https://github.com/ghiscoding/Angular-Slickgrid/releases). 

Please upvote ⭐ if you haven't already. 
Cheers
